### ISSUE TITLE
Incompatible with Webtrees 2.1.0

### ISSUE THREAD

**Issue by Neriderc** [JustCarmen/webtrees-fancy-research-links#56]
Webtrees 2.1.0 has been released, and this module doesn't appear to be compatible. It complains about a declaration not being compatible in the file FancyResearchLinksModule.php.

If I go into this file and comment out this section:

```
public function sidebarTitle(): string
    {
        return I18N::translate('Research links');
    }
```

Then it returns to working normally. However, I presume this breaks thinks if you are using a non-English language so it probably needs a proper fix.

**Comment by JustCarmen** [comment]
Be patiënt. Wt 2.1 had just been released. I Need some time tot update the code.

**Comment by Neriderc** [comment]
Sorry I'm not trying to rush you, mine is working fine after the above change so I'm in no hurry. Just raising an issue in case you weren't aware, and to help others that need a quick fix while waiting for a proper update.

**Comment by JustCarmen** [comment]
My apologies. I wrote this answer while I was in a hurry on my mobile phone (in a Dutch direct way). I appreciate your findings. This certainly helps to update the code for use with webtrees 2.1.

**Comment by Manja-S** [comment]
Could this problem be related to [issue #4328 in Webtrees](https://github.com/fisharebest/webtrees/issues/4328)?

**Comment by JustCarmen** [comment]
I've just released version 2.2.0 which is compatible with webtrees 2.1.

https://github.com/JustCarmen/webtrees-fancy-research-links/releases/latest
